creator-node/src/apiHelpers.js

```diff
@@ -66,21 +67,21 @@ module.exports.handleResponseWithHeartbeat = (func) => {
 
 const sendResponse = (module.exports.sendResponse = (req, res, resp) => {
   const duration = getDuration(req)
-  let logger = setFieldsInChildLogger(req, {
+  let childLogger = setFieldsInChildLogger(req, {
```

Comment:
I didn't like this pattern too much. Do please let me know if there's a better pattern for me to follow.

Reply:
@SidSethi that looks good, thanks!

I was referring to how to get line 22/23 to be `logger` instead of `console`.

If I import `logger` on line 4, then line 70 would be overwriting the global `logger` for the local `logger`. If you're okay with that approach, so am I. Thoughts?

Reply:
yeah i think thats fine to overwrite in local scope
can do either that or name this genericLogger, whichever

Reply:
Ok, good to know for next time, thanks! I'll use `genericLogger` today to match with the current format.